src/js/control-bar/mute-toggle.js

```diff
@@ -81,11 +81,11 @@ class MuteToggle extends Button {
     const vol = this.player_.volume();
     let level = 3;
 
-    if (vol === 0 || this.player_.muted()) {
+    if (this.player_.muted()) {
```

Comment:
what about only making this change?
I think level 1 is supposed to imply that the volume is very low and not just 0. So, if muted, level 0, if <0.33, level 1, if less than .67 level 2, else level 3?